sr_robot_launch/src/sr_robot_launch/mock_sr_ur_robot_hw.py

```diff
@@ -0,0 +1,260 @@
+                                               GetRobotMode, self.handle_get_robot_mode)
+        is_program_running = rospy.Service(self.arm_prefix + '_sr_ur_robot_hw/dashboard/program_running',
+                                           IsProgramRunning, self.handle_is_program_running)
+        load_program_service = rospy.Service(self.arm_prefix + '_sr_ur_robot_hw/dashboard/load',
+                                             Load, self.handle_load_program)
+        program_state_service = rospy.Service(self.arm_prefix + '_sr_ur_robot_hw/dashboard/program_state',
+                                              GetProgramState, self.handle_get_program_state)
+        power_on_service = rospy.Service(self.arm_prefix + '_sr_ur_robot_hw/dashboard/power_on',
+                                         Trigger, self.handle_power_on)
+        power_off_service = rospy.Service(self.arm_prefix + '_sr_ur_robot_hw/dashboard/power_off',
+                                          Trigger, self.handle_power_off)
+        brake_release_service = rospy.Service(self.arm_prefix + '_sr_ur_robot_hw/dashboard/brake_release',
+                                              Trigger, self.handle_brake_release)
+        restart_safety_service = rospy.Service(self.arm_prefix + '_sr_ur_robot_hw/dashboard/restart_safety',
+                                               Trigger, self.handle_restart_safety)
+        close_safety_popup_service = rospy.Service(self.arm_prefix + '_sr_ur_robot_hw/dashboard/close_safety_popup',
+                                                   Trigger, self.handle_close_safety_popup)
+        close_popup_service = rospy.Service(self.arm_prefix + '_sr_ur_robot_hw/dashboard/close_popup',
+                                            Trigger, self.handle_close_popup)
+        unlock_protective_stop_service = rospy.Service(self.arm_prefix +
+                                                       '_sr_ur_robot_hw/dashboard/unlock_protective_stop',
+                                                       Trigger, self.handle_unlock_protective_stop)
+        resend_robot_program_service = rospy.Service(self.arm_prefix + '_sr_ur_robot_hw/resend_robot_program',
+                                                     Trigger, self.handle_resend_robot_program)
+
+    def reinitialize(self):
+        self.robot_state = STATE()
+
+    def handle_get_safety_mode(self, request):
+        return self.robot_state.get_safety_mode()
+
+    def handle_get_program_state(self, request):
+        return self.robot_state.get_program_state()
+        response = GetProgramStateResponse()
+        if 'RUNNING' in self.state:
+            response.state = ProgramState.RUNNING
+        if 'PAUSED' in self.state:
+            response.state = ProgramState.RUNNING
+        if 'STOPPED' in self.state:
+            response.state = ProgramState.STOPPED
+        return response
```

Comment:
Won't this be skipped because of the return above?

Reply:
yup, good spot, thanks

Reply:
fixed here bee2ae4